### PR TITLE
Partly fix #1

### LINKED ISSUE #1: Site reports "Yes" even without WiFi
On wired LAN and WAN connections, the site erroneously reports "Yes". Expected: "No".

### PR DESCRIPTION
**Pull request by jmrtsh**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <html style="color-scheme: light dark">
 <center>
-  <h1 style="font-size: 120px">Yes.</h1>
+  <h1 id="statusText" style="font-size: 120px">Yes.</h1>
```
Comment (aengelke) [inline index.html]: Default to "Maybe.", s.t. no incorrect information is shown without JavaScript.

File: index.html [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+  <script>
+      if(navigator.connection.type == "wifi") {
```
Comment (aengelke) [inline index.html]:
```suggestion
      if((navigator.connection || {}).type === "wifi") {
```
For browsers without `navigator.connection` :shrug:

Comment (jmrtsh) [reply index.html]: Thanks!

**Review by aengelke — APPROVED** [review]
LGTM

**Review by kammt — APPROVED** [review]

**Comment by kammt** [comment]
This is why I love open-source

**Pull request merged**